src/main/java/discord4j/discordjson/json/ForumTagData.java

```diff
@@ -0,0 +1,32 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.Id;
+import discord4j.discordjson.possible.Possible;
+import org.immutables.value.Value;
+
+import java.util.Optional;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableForumTagData.class)
+@JsonDeserialize(as = ImmutableForumTagData.class)
+public interface ForumTagData {
+
+    static ImmutableForumTagData.Builder builder() {
+        return ImmutableForumTagData.builder();
+    }
+
+    Possible<Id> id();
+
+    Possible<String> name();
+
+    Possible<Boolean> moderated();
```

Comment:
Based in docs the moderated field exists in all cases then you can remove the Possible

Reply:
Resolved in 4e2c9e9b767324d7bbd6b51cf60d3611d0e57f61